src/main/scala/view/module/cactus/Panes.scala

```diff
@@ -138,7 +138,7 @@ class MainPane(controller: CactusController) extends ScalaFXPane:
    * @param player represented by the pane.
    * @param position of the pane.
    */
-  private class PlayerPane(player: Player, override val position: ViewPosition)
+  private class PlayerPane(player: Player, playerIndex: Int, override val position: ViewPosition)
```

Comment:
In the other part of the code I used `players.indexOf(player)` instead of using a value `playerIndex`.
If you consider that introducing a value is better than calculating the index everytime, change it in every occurrences.